content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+[2]: /api/#get-permissions
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_processors
+
+Grants a role the ability to create or modify the processors within a processing pipeline.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles page][1] and select the checkbox `write` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_write_processors_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+**Subset of Pipelines**:
+
+1. Remove the global permission on the role.
+2. This permission can be granted to a role in [the Processing Pipelines page of the Datadog app][2] by editing a processing pipeline and adding a role to the "Grant editing Processors of this index to" field (screenshot below).
+
+{{< img src="account_management/rbac/logs_write_processors.png" alt="Grant write access for processors to specific roles"  style="width:75%;" >}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+[2]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_archives
+
+Grants the ability to create or modify log archives.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_write_archives_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_public_config_api
+
+Grants the ability to create or modify log configuration through the Datadog API.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_public_config_api_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Data
+
+The following permissions can be granted to manage read access on subsets of log data:
+
+##### logs_read_index_data
+
+Grants a role read access on some number of log indexes. Can be set either globally or limited to a subset of log indexes.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles page][1] and select the checkbox `write` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_read_index_data_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+**Subset of Indexes**:
+
+1. Remove the global permission on the role.
```

Comment:
in this case, that means remove the `logs_read_index_data` permission and also remove the `logs_modify_indexes` permission.

I believe we want to make that explicit here, since it will reduce lots of pain and confusion for some users. And probably tickets too :)

Reply:
Fair point.

This is true up until we release `logs_read_data` which is coming up next week.